src/averageAges.js

```diff
@@ -35,6 +46,21 @@ function calculateMenAverageAge(people, century) {
  */
 function calculateWomenAverageAge(people, withChildren) {
   // write code here
+  let women = people.filter(person => person.sex === 'f');
+
+  women = (withChildren)
+    ? women.filter(person =>
+      people.some(woman =>
+        woman.mother === person.name
+      ))
+    : women;
+
+  const numberOfWomen = women.length;
+
+  const womenAverageAge = women.reduce((sum, currentValueWoman) =>
+    sum + currentValueWoman.died - currentValueWoman.born, 0);
```

Comment:
```suggestion
  const womenAverageAge = women.reduce((sum, currentValueWoman) =>
    sum + currentValueWoman.died - currentValueWoman.born, 0);
```
```suggestion
  const womenAverageAge = women.reduce((
    sum, currentValueWoman
  ) => sum + currentValueWoman.died - currentValueWoman.born, 0);
```
если надо перенести - переноси так